src/olympia/files/utils.py

```diff
@@ -206,20 +223,29 @@ def is_experiment(self):
         return any(bool(self.get(key)) for key in experiment_keys)
 
     @property
+    @raise_invalid_manifest_if_not_dict
+    def browser_specific_settings(self):
+        return self.get(
+            'browser_specific_settings', self.get('applications', EMPTY_FALLBACK_DICT)
+        )
+
+    @property
+    @raise_invalid_manifest_if_not_dict
     def gecko(self):
         """Return the "applications|browser_specific_settings["gecko"]" part
         of the manifest."""
-        parent_block = self.get(
-            'browser_specific_settings', self.get('applications', EMPTY_FALLBACK_DICT)
-        )
-        return parent_block.get('gecko', EMPTY_FALLBACK_DICT)
+        return self.browser_specific_settings.get('gecko', EMPTY_FALLBACK_DICT)
 
     @property
+    @raise_invalid_manifest_if_not_dict
     def gecko_android(self):
         """Return `browser_specific_settings.gecko_android` if present."""
-        return self.get('browser_specific_settings', EMPTY_FALLBACK_DICT).get(
-            'gecko_android', EMPTY_FALLBACK_DICT
-        )
+        return self.browser_specific_settings.get('gecko_android', EMPTY_FALLBACK_DICT)
+
+    @property
+    @raise_invalid_manifest_if_not_dict
+    def data_collection_permissions(self):
+        return self.gecko.get('data_collection_permissions', {})
```

Comment:
not new, but should this `{}` be `EMPTY_FALLBACK_DICT`?

Reply:
https://github.com/mozilla/addons-server/pull/24185/commits/423e1f8496e477e35bb48727de691cc92848ab2f